src/styles/bloks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+
+  &__image {
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+    box-sizing: border-box;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+    font-family: $main-fonts, sans-serif;
```

Comment:
Ensure that the variable `$main-fonts` is defined in your SCSS files. Without this definition, the SCSS will not compile correctly.